test/viewmodels/SecuritiesRequestedStatementsViewModelSpec.scala

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels
+
+import base.SpecBase
+import play.api.i18n.Messages
+import models.{EoriHistory, SecurityStatementsByPeriod, SecurityStatementsForEori}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.select.Elements
+import play.twirl.api.HtmlFormat
+import utils.Utils.{h2Component, spanLinkComponent}
+
+import java.time.LocalDate
+
+class SecuritiesRequestedStatementsViewModelSpec extends Setup {
+
+  when(mockMessages("cf.account.details.previous-eori", "EORI456"))
```

Comment:
Better to use messages(app: Application) (in SpecBase) so that the real msg keys can be tested. If you still need the mock then better to use inside the Setup trait as the tests should come after the class definitions as tests (description) are the first thing to look into